root/url/Header.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+// Copyright (C) 2017 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('../components/EntityTabs');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+
+type Props = {
+  url: URLT;
+  page: string;
+  entity_properties: EntityPropsT;
+};
+
+const URLHeader = ({url, page, entity_properties}: Props) => {
```

Comment:
`entity_properties` is a constant that should exist in `$c.stash` after you add it to this list: https://github.com/metabrainz/musicbrainz-server/blob/612cc7d/lib/MusicBrainz/Server.pm#L438

Constants don't need to be passed as props (since they never change, stateless functional components that reference them will remain pure). So every place that you either pass or accept `entity_properties` as a prop can be removed, and all places that read from `entity_properties` can instead read from `$c.stash.entity_properties`.

Reply:
Sorry, I forgot how our code works.

Don't change that line I linked, and don't access `$c.stash.entity_properties`. Just import `ENTITIES` from root/static/scripts/common/constants.js. :)